CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## UNRELEASED
+
+- INTERNAL: Upgrade dev dependencies
+- DOCS: Added documentation to repo
```

Comment:
@mirisuzanne Any additional changelog items we should add for this PR?

Reply:
We re-organized a few partials internally, and some of the docs – but no changes to documented features. Adding one note from that…

Reply:
K, not ready to document that as a public feature yet - but good to have it in the changelog. Should be good to go.